governance/default_test.go

```diff
@@ -549,6 +545,30 @@ func TestVoteValueNilInterface(t *testing.T) {
 		_, err := gov.ParseVoteValue(gVote)
 		assert.Equal(t, nil, err)
 	}
+
+	gVote.Key = "governance.addvalidator"
+	{
+		gVote.Value = [][]uint8{{0x3, 0x4}, {0x5, 0x6}}
```

Comment:
Don't we need to validate those values as if they were addresses?
Do we also allow "0x0304"?

Reply:
Yes, we allow 0x0304 at vote api. It is converted to `0x0000000000000000000000000000000000000304` while processing vote api(`adjustValueType`).
e.g.
```
> governance.vote("governance.addvalidator", "0x0304")
"Your vote was successfully placed."
> istanbul.getValidators()
["0x0000000000000000000000000000000000000304", "0x00000000000003233494f27b1cae261197d18f54", ...]
> 
```

So, as you suggested, ParseVoteValue should not accept 0x0304. If you agree with it, I'll add check logic and change the test case's results. @kjhman21

Reply:
@yoomee1313 In that case, it is okay, but we also have test cases for valid addresses. (I'm not sure we have it, but it does not show in this PR review.)

Reply:
@kjhman21 I'm sorry, the `adjustValueType` checking logic was wrong. Vote api should not accepts non-address length, too. I added address length checking logic to vote api and parseVoteValue. It is reflected at https://github.com/klaytn/klaytn/pull/1110/commits/4b2028fb3a43a13c90bb2c448aba3ad9f91fae21.

Reply:
@yoomee1313 Shouldn't this be failed due to the wrong length of addresses?

Reply:
@kjhman21 oh, sorry for the confuse.. i mean `vote api` didn't accept non-address string. And this PR changed it to accept non-address length . Than, I reverted it, and also changed parseVoteValue not to accept non-address value.